integration/utils/tx_stats_tracker.go

```diff
@@ -0,0 +1,132 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"go.uber.org/atomic"
+
+	"github.com/jedib0t/go-pretty/table"
+	"github.com/openhistogram/circonusllhist"
```

Comment:
I will add prometheus metrics to the loader later, so that we can have the loader-specific grafana dashboard (for continuous testing.).

The problem w/ prometheus histograms is that they:
* require pre-specifying buckets before hand.
* aren't designed to easily get values out of the interface after you put them in.

Basically, if one needs to programmatically track percentiles than either LogLogHist or HdrHist are the only scalable options.

Reply:
TL;DR the goal here is to replace howmebrewed statistics functions with an off the shelf library.